src/utils/vc.js

```diff
@@ -144,7 +144,10 @@ export function checkCredentialContext(credential) {
 function hasDockRevocation(status) {
   const id = status[credentialIDField];
   if (status
-    && jsonld.getValues(status, credentialTypeField).includes(RevRegType)
+    && (
+      jsonld.getValues(status, credentialTypeField).includes(RevRegType)
+      || jsonld.getValues(status, credentialTypeField).includes(`/${RevRegType}`)
```

Comment:
What pattern is this regex used to match?

Reply:
this isnt regex

Reply:
to explain a bit more, in json-ld 2.0.2 this method will potentially return value with a slash `/` prepended, so we are checking for that. later json-ld lib doesn't do such a thing